class-workflow.php

```diff
@@ -372,6 +297,42 @@ public function update_all_module_options( $mod_name, $new_options ) {
 		$this->$mod_name->module           = $this->modules->$mod_name;
 		return update_option( $this->options_group . $mod_name . '_options', $this->modules->$mod_name->options );
 	}
+
+
+	/**
+	 * Collect all of the active post types
+	 *
+	 * @return array $post_types All of the post types that are 'on'
+	 */
+	public function get_supported_post_types(): array {
```

Comment:
This function was moved out of the `Module` base class, which just called back into `VIP_Workflow` anyway.

This function relies on some filter timings related to older module registration (see `$this->settings->module->options->post_types` below) that would be tricky to refactor now. When we have moved all modules out of the old system, we can move this into a simpler static settings function.

A lot of code uses `get_supported_post_types()`, so this caused a lot of changes.